src/main/application-constants.ts

```diff
@@ -60,6 +60,12 @@ export function getCustomThemesPath() {
   return join(getConfigPath(), 'custom-themes')
 }
 
+// this is used for temporary files (because core expects file paths, can not accept blobs directly yet)
+// used when sending file from webxdc and when pasting a file from clipboard
+export function getDraftTempDir() {
+  return join(app.getPath('temp'), 'chat.delta.desktop-draft')
+}
```

Comment:
it is about the cleanup its easier, also that temp folder is a global temp folder used by all apps (at least on macOS), not a app unique folder as you might think. also see https://github.com/deltachat/deltachat-desktop/issues/2927

Reply:
> also that temp folder is a global temp folder used by all apps (at least on macOS), not a app unique folder as you might think

k, so this is a mostly unrelated fix. maybe call the folder more neutral so that it can be used for other things (that may not be drafts in the future), maybe just `chat.delta.desktop`. but again, no blocker